src/react-styleguide/src/components/Buttons/Button.js

```diff
@@ -27,7 +31,10 @@ const outlineBtnClasses = {
   default: 'b--white white hover-dark-navy hover-bg-white'
 }
 
-const Button = ({href, type='default', size='large', outline=false, children}) => {
+const Button = ({href, type='default', size='large', options=[], children}) => {
+    
+    const outline = options.includes('outline')
```

Comment:
Can remove this line once `outline` is a boolean, so it will work already with the ternary on line 38